experiments/README.md

```
@@ -0,0 +1,30 @@
+        --project=canada-spends \
+        --vercel-json=vercel.json \
+        --public \
+        --install datasette-chatgpt-plugin 
+```
+
+
+# TODO
+
+ - [ ] Proper Sourcing
+ - [ ] Proper Sourcing
+ - 
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)